<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="setting_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>95</width>
       <height>205</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="open_bottom">
        <property name="text">
         <string>Open file</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="run_step_bottom">
        <property name="text">
         <string>Run by step</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="save_bottom">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="stop_bottom">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="run_all_bottom">
        <property name="text">
         <string>Run all</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="close_bottom">
        <property name="text">
         <string>Close file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="sim_frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>280</y>
      <width>1151</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
   <widget class="QFrame" name="reg_frame">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>331</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QTableWidget" name="registerShow">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>11</y>
       <width>231</width>
       <height>261</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <row>
      <property name="text">
       <string>X0</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X19</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X20</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X21</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X22</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X23</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X24</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X25</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X26</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X27</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X28 (SP)</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X29 (FP)</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X30 (LR)</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>X31 (XZR)</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>value</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="c_flag">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: lightgray;
border: 2px solid black;
border-radius: 6px;
font-weight: bold;
font-size: 16px;
qproperty-alignment: AlignCenter;
</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QLabel" name="n_flag">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: lightgray;
border: 2px solid black;
border-radius: 6px;
font-weight: bold;
font-size: 16px;
qproperty-alignment: AlignCenter;
</string>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <widget class="QLabel" name="z_flag">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: lightgray;
border: 2px solid black;
border-radius: 6px;
font-weight: bold;
font-size: 16px;
qproperty-alignment: AlignCenter;
</string>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="v_flag">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: lightgray;
border: 2px solid black;
border-radius: 6px;
font-weight: bold;
font-size: 16px;
qproperty-alignment: AlignCenter;
</string>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="code_frame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>691</width>
      <height>281</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>671</width>
       <height>261</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
